public/css/style.css

```diff
@@ -0,0 +1,9 @@
+.register-text{
+    color: #0e0d0d;
+}
+.warning-text{
+    color: rgb(#ff6525);
```

Comment:
Blocker: when using rgb as value, it should have 3 arguments. Here you're using a hex color definition as argument. Either remove the wrapping `rgb` or define the color as rgb.